crates/holochain_conductor_config/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "holochain_conductor_config"
 version = "0.5.0-dev.0"
+description = "Provides utilities for generating holochain conductor configuration."
```

Comment:
Different order to most crates, want to be able to find the description quickly and easily when I open a manifest